src/Config/Config.php

```diff
@@ -46,9 +52,12 @@ private function getDefaultConfig(): array
     private function applyConfigFileValuesToConfig(array $config, ?string $configFilePath = null): array
     {
         if (!$configFilePath || !is_file($configFilePath)) {
-            return $config;
+            // fallback to /etc/tcms.conf
+            $configFilePath = sprintf('/etc/%s', trim(self::DEFAULT_CONF_FILENAME, '.'));
+            if (!is_file($configFilePath)) {
+                return $config;
+            }
         }
```

Comment:
It will, but first we will search environment variables for configuration and after that the method stopIfEmptyConfigParameter will trigger error which displays missing configurations.